mava/wrappers/shared.py

```diff
@@ -115,10 +115,12 @@ def observation_spec(
     ) -> Union[specs.Spec[Observation], specs.Spec[ObservationGlobalState]]:
         """Specification of the observation of the `RobotWarehouse` environment."""
         obs_spec = self._env.observation_spec()
-        num_obs_features = obs_spec.agents_view.shape[-1] + self._env.num_agents
+        num_obs_features = obs_spec.agents_view.shape[-1]
```

Comment:
Nice so this means we don't expect to repeat the agent_ids in the global state, right?

Reply:
These changes support both methods of adding all agent_ids to the global state or just one. But for now, we've fixed it here to add only one ID per agent as usual in the global state (this is tested with both approaches here #965, we'll see what to fix after finishing this investigation).